geth/node/status_node.go

```diff
@@ -282,81 +302,74 @@ func (n *StatusNode) removePeer(url string) error {
 	if err != nil {
 		return err
 	}
+
+	if !n.isRunning() {
+		return ErrNoRunningNode
+	}
+
 	n.gethNode.Server().RemovePeer(parsedNode)
+
 	return nil
 }
 
 // PeerCount returns the number of connected peers.
 func (n *StatusNode) PeerCount() int {
-	if !n.IsRunning() {
-		return 0
-	}
-	return n.gethNode.Server().PeerCount()
-}
-
-// Config exposes reference to running node's configuration
-func (n *StatusNode) Config() (*params.NodeConfig, error) {
 	n.mu.RLock()
 	defer n.mu.RUnlock()
 
-	if err := n.isAvailable(); err != nil {
-		return nil, err
+	if !n.isRunning() {
+		return -1
```

Comment:
This not standard behaviour should probably be documented at the method comment.

Reply:
I changed it to `0`.